tickets_cli/src/appointment/send.rs

```diff
@@ -70,13 +89,12 @@ pub async fn send_invitation_letter_handler(
         email
     };
 
-    let mut sorted_emails = email_list;
-    sorted_emails.sort();
-    sorted_emails.dedup();
+    let unique_emails: Vec<_> = email_list
+        .into_iter().collect::<HashSet<_>>().into_iter().collect();
```

Comment:
Better leave it as it were.

Creating a HashSet and then again creating a Vec is not a good idea of introducing additional allocations... :)